lib/api.js

```diff
@@ -61,10 +61,10 @@ function getGist(uid) {
       const snippet = files[otherFiles[0]]
 
       return {
+        gist: true,
         code: snippet.content,
-        language: snippet.language,
-        owner,
-        config
+        ...(snippet.language ? { language: snippet.language.toLowerCase() } : {}),
+        ...(typeof config === 'object' ? { config } : {})
```

Comment:
I think the ordering here should be: 
```js
{
  ...(typeof config === 'object' ? { config } : {}),
  ...(snippet.language ? { language: snippet.language.toLowerCase() },
  code: snippet.content,
  gist: true
}
```

I also think it was a lot more clear before when we mutated the object, instead of messing around with all these `...`'s (i know I typically like those better 🙂 )